test/tests/damage_plasticity_model/tests

```diff
@@ -0,0 +1,57 @@
+[Tests]
+  issues = '#184'
+  design = 'DamagePlasticityStressUpdate.md ComputeMultipleInelasticDamageStress.md'
+  [concrete_behavior]
+    requirement = 'Blackbear shall demonstrate behavior of concrete '
```

Comment:
I'd be more descriptive:
`BlackBear shall provide a damaged plasticity model for concrete, which demonstrates the desired behavior `